app/controllers/api/v2/hosts_controller.rb

```diff
@@ -179,36 +179,17 @@ def enc
         render :json => { :data => @host.info }
       end
 
-      api :GET, "/hosts/:id/status", N_("Get configuration status of host")
-      param :id, :identifier_dottable, :required => true
-      description <<~EOS
-        Return value may either be one of the following:
-
-        * Alerts disabled
-        * No reports
-        * Error
-        * Out of sync
-        * Active
-        * Pending
-        * No changes
-      EOS
-
-      def status
-        Foreman::Deprecation.api_deprecation_warning('The /status route is deprecated, please use the new /status/configuration instead')
-        render :json => { :status => @host.get_status(HostStatus::ConfigurationStatus).to_label }.to_json if @host
-      end
-
       api :GET, "/hosts/:id/status/:type", N_("Get status of host")
       param :id, :identifier_dottable, :required => true
-      param :type, [ HostStatus::Global ] + HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_(<<~EOS
-        status type, can be one of
-        * global
-        * configuration
-        * build
-      EOS
-  # rubocop:disable Layout/ClosingParenthesisIndentation
-)
-      # rubocop:enable Layout/ClosingParenthesisIndentation
+      param :type, [ HostStatus::Global ] + HostStatus.status_registry.to_a.map { |s| s.humanized_name }, :required => true, :desc => N_(
+        <<~EOS
+          status type, can be one of
+          * global
```

Comment:
Can we retrieve this from the status registry?

Reply:
IIRC we normally generate the apidoc during packaging, meaning that it won't catch any others anyway except for the rare few who regenerate the doc